src/lib/fhirTables.js

```diff
@@ -523,14 +537,15 @@ function carePlanRow(r, rmap, dcr) {
     ', '
   );
 
-  const category = futil.joinJSXElements(
-    (r.category || []).map(c =>
-      c.text ? futil.renderCodeableJSX(c, dcr) : null
-    ).filter(c => c !== null),
-    ', '
-  );
+   const category = futil.joinJSXElements(
+       (r.category || []).map(c =>
+          futil.renderCodeableJSX(c, dcr)
+       ).filter(c => c !== null),
+       ', '
+   );
 
   const period = r.period ? futil.renderPeriod(r.period) : "";
+  const note = r.note ? r.note : "";
```

Comment:
I think this needs to be `r.note ? r.note.text : ""` --- the note itself is an "Annotation" which always has at least the text field required.

Reply:
You're correct -- updated this line! Good catch.